src/display/editor/tools.js

```diff
@@ -279,6 +280,63 @@ class ClipboardManager {
   }
 }
 
+class ColorManager {
+  static _colorsMapping = new Map([
+    ["CanvasText", [0, 0, 0]],
+    ["Canvas", [255, 255, 255]],
+  ]);
+
+  get _colors() {
+    if (typeof document !== "undefined") {
+      const colors = new Map([
+        ["CanvasText", null],
+        ["Canvas", null],
+      ]);
+      getColorValues(colors);
+      return shadow(this, "_colors", colors);
+    }
+
+    return shadow(this, "_colors", ColorManager._colorsMapping);
+  }
+
+  /**
+   * In High Contrast Mode, the color on the screen is not always the
+   * real color used in the pdf.
+   * For example in some cases white can appear to be black but when saving
+   * we want to have white.
+   * @param {string} color
+   * @returns {Array<number>}
+   */
+  convert(color) {
+    const rgb = getRGB(color);
+    if (!window.matchMedia("(forced-colors: active)").matches) {
+      return rgb;
+    }
+
+    for (const [name, RGB] of this._colors) {
+      if (RGB.every((x, i) => x === rgb[i])) {
+        return ColorManager._colorsMapping.get(name);
+      }
+    }
+    return rgb;
+  }
+
+  /**
+   * An input element must have its color value as a hex string
+   * and not as color name.
+   * So this function converts a name into an hex string.
+   * @param {string} name
+   * @returns {string}
+   */
+  getHexCode(name) {
+    const rgb = this._colors.get(name);
+    if (!rgb) {
+      return name;
+    }
+    return "#" + rgb.map(x => x.toString(16).padStart(2, "0")).join("");
```

Comment:
Wait, isn't this *identical* to the existing `Util.makeHexColor` method?
Please use that one instead, rather than reinventing the wheel here :-)

Reply:
Especially when I created this wheel myself...